examples/stable_diffusion.py

```diff
@@ -636,6 +638,8 @@ def do_step(latent, timestep, index, guidance):
   if args.seed is not None: Tensor._seed = args.seed
   latent = Tensor.randn(1,4,64,64)
 
+  BEAM.value = getenv("BEAM")
+
   # this is diffusion
```

Comment:
I would do this as LATEBEAM like how I do it in compile2

`with Context(BEAM=getenv("LATEBEAM"):`